src/main/java/net/pms/encoders/AviSynthFFmpeg.java

```diff
@@ -81,7 +81,7 @@ public String initialString() {
 		if (configuration.isFfmpegAviSynthMultithreading()) {
 			threads = " -threads " + configuration.getNumberOfCpuCores();
 		}
-		return configuration.getMPEG2MainSettingsFFmpeg() + " -ab " + configuration.getAudioBitrate() + "k" + threads;
+		return configuration.getMPEG2MainSettingsFFmpeg() + " -ab " + configuration.getAudioBitrateAC3() + "k" + threads;
```

Comment:
@SubJunk are you sure that this is correct? What about if the `audio_bitrate_ac3` will be set `Automatic` the result will be e.g. ` -ab Automatick -threads 2`